app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java

```diff
@@ -251,7 +251,8 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             return;
         if (requestCode == INTENT_REQUEST_PICKFILE_CODE) {
             //Getting Absolute Path
-            String path = RealPathUtil.getInstance().getRealPath(getContext(), data.getData());
+            Uri uri = data.getData();
```

Comment:
how is this fixing #759?

Reply:
Change in get an absolute path file. Now, get an absoluth path from URI. 

The original solution gave an inccorect path. The app could not access the file. Check photo. File name is not visible and preview generates an error.

I tested on 2 phone. 1 phone (Xiaomi) needed a new external file explorer (ES File Manager). 
![Screenshot_2019-11-02-09-20-57-135_swati4star createpdf](https://user-images.githubusercontent.com/22313012/68068179-ea350680-fd52-11e9-993c-41760468b0c4.png)

Reply:
The issue described is different.
What your are solving is something else.

Please share the File path for which you are facing this error. And fix it in realpathutils. You are fixing it in incorrect way